### PR TITLE
feat: [CHK-3217] Upgrade helm chart + autoscaling limits

### PR DESCRIPTION
**Pull request by petretiandrea**
This PR implements the results of soak test focused on horizontal autoscaling.

See:
- [Soak con delay](https://pagopa.atlassian.net/wiki/spaces/I/pages/1241284669/Soak+con+delay+mock+Esecuzione+06+09+2024+-+esecuzione+1)
- [Soak](https://pagopa.atlassian.net/wiki/spaces/I/pages/1058963457/Soak+Esecuzione+15+05+2024+-+esecuzione+1)

#### List of Changes
- bump pagopa helm chart to 2.8.0 to supports `advanced` section of autoscaling
- increased CPU limits to 900m to reduce JVM warmup time and K8S throttling. Also provide better response time when requests hits a cold PODs
- added `stabilizationWindows` to scaleUp behaviour in order to reduce scaleup flapping behavior

#### Motivation and Context
Achieve better system stability (in terms of response time and failures) during scale up & scale down phases.

#### How Has This Been Tested?
The only one breaking change is related to `handlerType` key on K8S health check section. 

I've tested it on DEV by:
1. deploying transaction-service (pod becomes ready)
2. checking environment variables
3. making a payment (cards and apm)
4. a comparison between two helm chart version [diff](https://github.com/pagopa/aks-microservice-chart-blueprint/compare/microservice-chart-2.3.1..microservice-chart-2.8.0)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by giovanniberti — APPROVED** [review]

**Review by pietro-tota — DISMISSED** [review]

File: helm/Chart.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: microservice-chart
   repository: https://pagopa.github.io/aks-microservice-chart-blueprint
-  version: 2.3.1
-digest: sha256:92b42c64847373faa6bf054181be2d7ee61e8b6b6d6a5552bf1479691aa95df3
-generated: "2023-01-27T15:52:58.429288+01:00"
+  version: 2.8.0
```
Comment (infantesimone) [inline helm/Chart.lock:4]: the update to version 2.8.0 is safe and has no side effects?

**Review by infantesimone — APPROVED** [review]
are we sure that the update to version x is safe and has no side effects?

File: helm/Chart.lock (hunk 1 above)
Comment (infantesimone) [reply helm/Chart.lock:4]: great!

**Comment by infantesimone** [comment]
pay attention to code-review fails

**Review by giovanniberti — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**